### PR TITLE
Adding support to parsing single object to a Feature

### PR DESCRIPTION
**Pull request by herkit**
I needed to be able to store Features in a database as single entities for later concatenation into a FeatureCollection, however this module forces Features into a FeatureCollection. I added a simple check if the data parameter is an array or not to allow conversion to a single non-collection wrapped Feature.

### CODE REVIEW

**Comment by caseypt** [comment]
Thanks for the PR! I'll take a look at this soon.

File: geojson.js [hunk 1]
```diff
@@ -8,19 +10,23 @@
...
     geomAttrs.length = 0; // Reset the list of geometry fields
     setGeom(settings);
     propFunc = getPropFunction(settings);
 
-    objects.forEach(function(item){
-      geojson.features.push(getFeature(item, settings, propFunc));
-    });
-
-    addOptionals(geojson, settings);
+    if (util.isArray(objects)) {
```
Comment (caseypt) [inline geojson.js]: Consider swapping this out for `Array.isArray`. `require`ing a module will break this library for vanilla JavaScript in the browser. Looking at the [source](https://github.com/nodejs/node/blob/master/lib/util.js#L834) of `util.isArray`, it's just a direct export of `Array.isArray`, so the code should be functionally the same.

**Comment by caseypt** [comment]
I left a comment for your consideration. Thanks again for the PR!

**Comment by herkit** [comment]
Hi, I changed as per your comment, how Array.isArray missed my attention is beyond me.

**Comment by caseypt** [comment]
Thanks again for the PR. Sorry about the delay to get this in. Merging now.

**Comment by caseypt** [comment]
I just pushed v0.4.0 to npm, which includes this change. Thanks again.

**Pull request merged**